api/src/main/java/org/pmiops/workbench/db/dao/DataSetServiceImpl.java

```diff
@@ -496,11 +497,16 @@ private boolean supportsConceptSets(Domain domain) {
 
   private QueryJobConfiguration buildQueryJobConfiguration(
       Map<String, QueryParameterValue> namedCohortParameters, String query) {
+    return QueryJobConfiguration.newBuilder(query)
```

Comment:
I can do the function that returns a builder with the arg set, that seems reasonable to me. Because its in two places, I would prefer having the default args change in both with one change.

Reply:
Talked offline: 
I tried to address the comment here https://github.com/all-of-us/workbench/pull/3103/files#diff-5324f051d4e486d6f33e30b9f6bebd92R500 but it looks like all the builder methods that don't require a query up front are private methods. :cry:
I could have it build each time it is used, but that ends up with a bunch of repeated boilerplate for the defaults.
11:20
(I could maybe build with an empty string literal and then set the query later?) (edited) 
11:20
Do you have a preference?

Jay Carlton  11:26 AM
it's not a huge win to do what I think I was saying





11:26
just pointing out that builders smaller than ~5 lines are all right inline unless there's lots of repeated fields